docs/compose/FloatingActionButton/README.md

```diff
@@ -0,0 +1,21 @@
+# Floating Action Button
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+Example of a `BpkFab`:
+
+```Kotlin
+import net.skyscanner.backpack.compose.button.BpkFab
+
+BpkFab(
+  onClick = { BpkToast.makeText(context, "Message", BpkToast.LENGTH_SHORT).show() },
```

Comment:
No need to use toast in the example, you can use empty lambda here. You can check out the other docs for Compose components for reference